source/_drafts/Video-js-7-is-here.md

```diff
@@ -0,0 +1,79 @@
+This release is pretty exciting because it means we were able to [remove a bunch of code](https://github.com/videojs/video.js/pull/5041) for old browsers and we can now start using newer web platform features that we had previously been avoiding like the `:not()` css selector. It'll also free us to look forward at new features and not be held back by needing to support old browsers like IE8.
+
+## Getting Video.js 7
+
+You can get it from npm via
+```sh
+npm install video.js@next
+```
+or
+```sh
+npm install video.js@7.0.0
+```
+
+From [GitHub Releases](https://github.com/videojs/video.js/releases/tag/v7.0.0),
+
+Or from our or another CDN
+```
+https://vjs.zencdn.net/7.0.0/video.min.js
+https://vjs.zencdn.net/7.0.0/video-js.css
+```
+```
+https://unpkg.com/video.js@7.0.0/dist/video.min.js
+https://unpkg.com/video.js@7.0.0/dist/video-js.css
+```
+```
+https://cdnjs.cloudflare.com/ajax/libs/video.js/7.0.0/video.min.js
+https://cdnjs.cloudflare.com/ajax/libs/video.js/7.0.0/video-js.css
+```
+
+## VHS and Video.js
+Video.js will be bundling VHS by default for ease of use for new users. However, some people don't want VHS or are using another plugin. For this, we have a separate build of Video.js which doesn't include VHS.
+You can get it from our CDN or another CDN
+```
+https://vjs.zencdn.net/7.0.0/alt/video.core.min.js
+```
+```
+https://unpkg.com/video.js@7.0.0/dist/alt/video.core.min.js
+```
+```
+https://cdnjs.cloudflare.com/ajax/libs/video.js/7.0.0/alt/video.core.min.js
+```
+
+Finally, expect a more details post about VHS in the weeks to come.
+
+## What the future holds
+Video.js hasn't really had a roadmap for a long time. With this release, we think it's time to take a step back and create a roadmap.
+The roadmap will take into account currently open issues and internal and external feature requests. In addition, we're going to look at currently available features and see if it's worth refactoring them with the more modern web in-mind.
+Later down the line, we'd also want to make a roadmap for a Video.js 8. Given that this release only adds VHS and removes old browsers, there's plenty of already deprecated features that it's time to remove.
+
+## Updates from the [Video.js 7 roadmap][roadmap]
+### Video.js 5.x
+
+As of this release, Video.js 5.x is now officially deprecated. This means that we will no longer work on it or back-port fixes to it but it's still available for usage if you're still using it. We do urge you to update to the latest 5.x if you're on it, or better yet, upgrade to 6.x or 7 if you can.
+
+### Build Tools
+In the [roadmap][] post, we talked about potentially switching to webpack from rollup for better support of importing VHS into Video.js. However, while doing [this work](https://github.com/videojs/video.js/pull/5033), we found out that the filesize of Video.js will increase by about 20%, which isn't a trivial amount, not to mention that the filesize is already going to increase from the default inclusion of VHS.
```

Comment:
Missing roadmap link? Also, we should probably capitalize Rollup since it's capitalized on their website. Webpack is probably fine as-is since that's their "branding".

Reply:
it's a shorthand for the bottom links thing, since the reference will is also `roadmap`, you don't need to double it.